extra/2-piping.js

```diff
@@ -16,26 +16,30 @@
   the final result to the variable goodCode
 */
 
-function add() {
-
+function add(input1, input2) {
+let addTotal = input1 + input2 ;
+return addTotal ;
 }
 
-function multiply() {
-
+function multiply(Num1, Num2) {
```

Comment:
It looks like this code will work 👍 
Small point: when we are writing code, we usually try and follow some conventions - like parameter names should start with a lowercase letter. Following conventions like this can help make our code easier to read for other developers.
You can see more JavaScript conventions here: https://www.30secondsofcode.org/articles/s/javascript-naming-conventions/